app/views/JitsiMeetView/index.tsx

```diff
@@ -71,25 +97,49 @@ const JitsiMeetView = (): React.ReactElement => {
 		};
 	}, [handleJitsiApp, onConferenceJoined, videoConf]);
 
+	useEffect(() => {
+		setCookies();
+	}, []);
+
 	const callUrl = `${url}${url.includes('#config') ? '&' : '#'}config.disableDeepLinking=true`;
 
 	return (
-		<SafeAreaView style={{ flex: 1 }}>
+		<SafeAreaView style={styles.container}>
 			{authModal && <JitsiAuthModal setAuthModal={setAuthModal} callUrl={callUrl} />}
-			<WebView
-				source={{ uri: callUrl.replace(/"/g, "'") }}
-				onNavigationStateChange={onNavigationStateChange}
-				onShouldStartLoadWithRequest={onNavigationStateChange}
-				style={{ flex: 1, backgroundColor: 'rgb(62,62,62)' }}
-				userAgent={userAgent}
-				javaScriptEnabled
-				domStorageEnabled
-				allowsInlineMediaPlayback
-				mediaCapturePermissionGrantType={'grant'}
-				mediaPlaybackRequiresUserAction={isIOS}
-			/>
+			{cookiesSet ? (
+				<WebView
+					source={{
+						uri: callUrl.replace(/"/g, "'"),
+						headers: {
+							Cookie: `rc_uid=${user.id}; rc_token=${user.token}`
+						}
+					}}
+					onNavigationStateChange={onNavigationStateChange}
+					onShouldStartLoadWithRequest={onNavigationStateChange}
+					style={styles.webviewContainer}
+					userAgent={userAgent}
+					javaScriptEnabled
+					domStorageEnabled
+					allowsInlineMediaPlayback
+					mediaCapturePermissionGrantType={'grant'}
+					mediaPlaybackRequiresUserAction={isIOS}
+					sharedCookiesEnabled
+				/>
+			) : (
+				<View style={[styles.webviewContainer, styles.loading]}>
+					<ActivityIndicator />
+				</View>
+			)}
 		</SafeAreaView>
 	);
 };
 
+const styles = StyleSheet.create({
+	container: {
+		flex: 1
+	},
+	webviewContainer: { flex: 1, backgroundColor: 'rgb(62,62,62)' },
+	loading: { alignItems: 'center', justifyContent: 'center' }
+});
```

Comment:
I understand that the prettier don't see the necessity to break the line, but can you break them?